docs/source/intro/overview.rst

```diff
@@ -212,6 +212,25 @@ Agent can run both from container and from a local machine.
 
         python3 -m core.run -ch telegram
 
+    or via HTTP api client:
+    .. code:: bash
+        python3 -m core.run -ch http_client [-p 8888]
+
+    HTTP client accepts POST requests with application/json content-type.
+    Request should be in form:
+    .. code:: json
+        {
+          "user_id": "unique id of user"
+          "payload": "phrase, which should be processed by agent"
+        }
+    Agent will return a json response:
+    .. code:: json
+        {
+          "user_id": "same user id as in request"
+          "response": "phrase, which were generated by skills in order to respond"
+        }
+    In case of wrong format, HTTP errors will be returned
+
```

Comment:
Would be nice to add to documentation a `curl` example

Reply:
btw, when tested with `curl`, we get funny results:
`{"user_id": 1, "response": "\u041f\u0440\u0438\u0432\u0435\u0442 \u0435\u0449\u0451 \u0440\u0430\u0437"}%`
Maybe it can be easily fixed.